README.md

```diff
@@ -1,30 +1,41 @@
-FSharpPlus [![Build Status](https://api.travis-ci.org/fsprojects/FSharpPlus.svg?branch=master)](https://travis-ci.org/fsprojects/FSharpPlus) [![Build status](https://ci.appveyor.com/api/projects/status/25ukpc0lamyf7pdx/branch/master?svg=true)](https://ci.appveyor.com/project/wallymathieu/fsharpplus/branch/master) [![Join the chat at https://gitter.im/fsprojects/FSharpPlus](https://badges.gitter.im/fsprojects/FSharpPlus.svg)](https://gitter.im/fsprojects/FSharpPlus?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+FSharpPlus
+[![Build Status](https://api.travis-ci.org/fsprojects/FSharpPlus.svg?branch=master)](https://travis-ci.org/fsprojects/FSharpPlus) 
+[![Build status](https://ci.appveyor.com/api/projects/status/25ukpc0lamyf7pdx/branch/master?svg=true)](https://ci.appveyor.com/project/wallymathieu/fsharpplus/branch/master) 
+[![Downloaded](https://img.shields.io/nuget/dt/FSharpPlus.svg)](https://www.nuget.org/packages/FSharpPlus)
+[![NuGet](https://img.shields.io/nuget/v/FSharpPlus.svg)](https://www.nuget.org/packages/FSharpPlus)
+[![NuGet Pre](https://img.shields.io/nuget/vpre/FSharpPlus.svg?label=pre)](https://www.nuget.org/packages/FSharpPlus/absoluteLatest)
 ==========
 
-A complete and extensible base library for F#.
+F#+ is a base library that aims to take F# to the next level of functional
+programming. 
 
-It contains the most requested additions to the F# core library, including:
+*What if we imagine F# as more than it is?*
 
- - Common FP combinators, generic functions and operators.
- - Extension methods for F# types with consistent names and signatures.
- - Standard Monads: Cont, Reader, Writer, State and their Monad Transformers.
- - Other popular [FP abstractions](//fsprojects.github.io/FSharpPlus/abstractions.html).
- - [Generic Functions and Operators](//fsprojects.github.io/FSharpPlus/reference/fsharpplus-operators.html) which may be further extended to support other types.
- - Generic and customizable [Computation Expressions](//fsprojects.github.io/FSharpPlus/computation-expressions.html).
- - A [generic Math](//fsprojects.github.io/FSharpPlus/numerics.html) module.
- - A true polymorphic [Lens/Optics module](//fsprojects.github.io/FSharpPlus/tutorial.html#Lens).
```

Comment:
This is all down in index page now, with slightly more new-user friendly language